src/Form/MetadataDisplayForm.php

```diff
@@ -174,17 +175,54 @@ public function save(array $form, FormStateInterface $form_state) {
     return $status;
   }
 
+  public static function flattenKeys(array $array, string $recursive_key = '', int $array_depth = 0, $excludepaths = []) {
+    $return = [];
+    $array_depth_max = 10;
+    ++$array_depth;
+    // The below is copied from StrawberryfieldJsonHelper::arrayToFlatJsonPropertypaths to deal with arrays,
+    // but maybe for the next iteration, it would be better to add options to
+    // that existing function and import here.
+    // Blacklist paths. Strip the last dot in case this was called recursively.
+    if (!empty($excludepaths) && in_array(rtrim($recursive_key,'.'), $excludepaths)) {
+      return $flat;
+    }
+    foreach($array as $key=>$value) {
+      if(filter_var($key, FILTER_VALIDATE_URL) || StrawberryfieldJsonHelper::validateURN($key)) {
```

Comment:
Was `StrawberryfieldJsonHelper` included on the header as an USE statement? I don't see it

Reply:
https://github.com/esmero/format_strawberryfield/blob/2efe80497d9a336a4ef79846fa2bb1a52a7d4b27/src/Form/MetadataDisplayForm.php#L14

Did I miss an additional step?

Reply:
You good. Strange the Code preview/diff did not show it. Marked as resolved